packages/gestalt/src/icons/graph-bar.svg

```diff
@@ -1 +1 @@
-<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path fill="#8e8e8e" d="M1 23V10h5v13H1zm8.5 0V1h5v22h-5zm8.5 0V7h4.999v16H18z"/></svg>
\ No newline at end of file
+<svg height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="m13.5 1h-3c-.552 0-1 .448-1 1v20c0 .552.448 1 1 1h3c.552 0 1-.448 1-1v-20c0-.552-.448-1-1-1m8.5 6h-3c-.552 0-1 .448-1 1v14c0 .552.448 1 1 1h3c.552 0 1-.448 1-1v-14c0-.552-.448-1-1-1m-16 4v11c0 .552-.448 1-1 1h-3c-.552 0-1-.448-1-1v-11c0-.552.448-1 1-1h3c.552 0 1 .448 1 1" fill="#8e8e8e" fill-rule="evenodd"/></svg>
```

Comment:
This also doesnt look minimized or optimized by PDS folks
![image](https://user-images.githubusercontent.com/7877010/57333529-eee56e80-70d1-11e9-9919-6f1483547755.png)